recipes/torchhd/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "torch-hd" %}
+{% set version = "2.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4ffb7c627441aa71a3e286cab035bbe9c52d7ccfaf2605c613a22cdfea1eeccb
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
If the only reason is a missing dependency, this can continue to stay noarch.

Reply:
Yes, you can just specify PyTorch and ignore the failure on Windows. That way, if PyTorch ever becomes available on Windows, both packages can be installed.

Reply:
Right, I changed it to stay noarch in python. Also, How would I explicitly ignore the failure on Windows?

Reply:
Ahh, ignoring means literally that. We’ll just ignore it here. :smile:
In the feedstock, noarch packages are only built on Linux so you won‘t see the error anymore. 
Ready to get merged then?

Reply:
Alright, I understand now haha. Thank you!
I just made the final change and made it noarch.